src/Persistence/PaYaml/Serialization/PFxExpressionYamlConverter.cs

```diff
@@ -58,6 +58,8 @@ public void WriteYaml(IEmitter emitter, object? value, Type type)
         // Force multi-line scripts to be literal blocks
         forceLiteralBlock |= expression.InvariantScript.Contains('\n')
             || expression.InvariantScript.Contains('\r');
+        // Force literal block for Unicode characters to preserve them without escaping
+        forceLiteralBlock |= expression.InvariantScript.Any(c => c > 127 && !char.IsControl(c));
```

Comment:
Is the issue occurring for every character above 2F / 127?
For instance, is that occurring for the €, ¥, £ (Euro, Yen, British Pound) found in the 80 - FF (128-255) range?

Reply:
this is a good point. maybe we should have a more targeted conditional?